### PR TITLE
added basic marker block token rules

### PR DESCRIPTION
**Pull request by remo-help**
marker blocks now implemented as basic token rules

will expand with some dependency rules as soon as I get to it!

### CODE REVIEW

**Review by adarshp — APPROVED** [review]

File: src/main/resources/org/clulab/asist/grammars/items.yml [hunk 1]
```diff
@@ -41,3 +41,36 @@ rules:
...
       owner: Entity? = >"nmod:poss"
+
+# -------- marker blocks ----------
+  - name: marker_block
+    priority: ${ rulepriority }
+    label: MarkerBlock
+    type: token
+    keep: true
+    pattern: |
+      [lemma=/(?i)marker block|marker/]
```
Comment (BeckySharp) [inline src/main/resources/org/clulab/asist/grammars/items.yml:52]: This won't work, what's inside the `[ ]` has to be a single token.
maybe something like this?
```
[lemma=marker] [lemma=block]?
```

File: src/main/resources/org/clulab/asist/grammars/items.yml [hunk 2]
```diff
@@ -41,3 +41,36 @@ rules:
...
+
+  - name: critical_marker_token
+    priority: ${ rulepriority }
+    label: CriticalMarkerBlock
+    type: token
+    keep: true
+    pattern: |
+      [lemma=/(?i)critical|critical victim/] [lemma=/(?i)marker|marker block/]
```
Comment (BeckySharp) [inline src/main/resources/org/clulab/asist/grammars/items.yml:60]: same here:
```
[lemma=critical] [lemma=victim]? [lemma=marker] [lemma=block]?
```

File: src/main/resources/org/clulab/asist/grammars/items.yml [hunk 3]
```diff
@@ -41,3 +41,36 @@ rules:
...
+
+  - name: regular_marker_token
+    priority: ${ rulepriority }
+    label: RegularMarkerBlock
+    type: token
+    keep: true
+    pattern: |
+      [lemma=/(?i)regular|regular victim/] [lemma=/(?i)marker|marker block/]
```
Comment (BeckySharp) [inline src/main/resources/org/clulab/asist/grammars/items.yml:68]: same

File: src/main/resources/org/clulab/asist/grammars/items.yml [hunk 4]
```diff
@@ -41,3 +41,36 @@ rules:
...
+
+  - name: no_victim_marker_token
+    priority: ${ rulepriority }
+    label: NoVictimMarkerBlock
+    type: token
+    keep: true
+    pattern: |
+      [lemma=/(?i)no victim|empty/] [lemma=/(?i)marker|marker block/]
```
Comment (BeckySharp) [inline src/main/resources/org/clulab/asist/grammars/items.yml:76]: same

**Review by BeckySharp — CHANGES_REQUESTED** [review]
Thanks for working on this! Please see comments

File: src/main/resources/org/clulab/asist/grammars/items.yml (hunk 1 above)
Comment (remo-help) [reply src/main/resources/org/clulab/asist/grammars/items.yml:52]: thanks!

**Comment by remo-help** [comment]
@BeckySharp I put in the changes, but I cant merge for some reason?

**Review by BeckySharp — APPROVED** [review]
thanks! fyi the lemma  form is (pretty sure) always lowercase

**Pull request merged**